11ty/definitions/biromantic.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Biromantic
+slug: biromantic
+speech: adj
+defined: true
+---
+
+of, relating to, or characterised by being romantically attracted to more than one gender.
+
+#### Note
+
+Biromantic does not imply any particular kind of sexual attraction, or sexual attraction at all. A biromantic person may be asexual, homosexual, heterosexual, [bisexual](#bisexual), [pansexual](#pan-sexual), etc.
+
+Biromantic does not preclude romantic attraction to [non-binary](#non-binary) or [transgender](#transgender) people.
+
+#### Further Reading
+
+[Sorting Out Romantic vs Sexual Feelings](http://http://teenhealthsource.com/blog/faq-sorting-out-romantic-vs-sexual-feelings/); [Being Bi On The Asexual Spectrum](https://www.curvemag.com/Lifestyle/Being-Bi-On-The-Asexual-Spectrum-2141/); [Here’s What It Means When Your Romantic and Sexual Orientations Are Different](https://everydayfeminism.com/2016/07/cross-orientation-101/)
```

Comment:
The last link isn't about "biromantic" specifically but it seemed relevant to me because "bisexual" tends to be more visible, and some people might confuse the terms. Maybe this should be split out into cross-orientation though.